src/test/common/steps/complete-flow.js

```diff
@@ -16,13 +16,10 @@ Given('I am on the first page of the application', async function () {
 })
 
 Given('I am on the check details page having entered valid details for a pregnant woman', async function () {
-  await pages.startApplication()
```

Comment:
All the steps removed in this file were superfluous which meant that extra page navigation being done for no reason. Similarly the page waits are now always done by the navigation functions.